packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,274 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import Anatomy from './ActionBarAnatomy.svg';
+import docs from 'docs:@react-spectrum/actionbar';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/actionbar/package.json';
+
+```jsx import
+import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
+```
+
+---
+category: Collection Actions
+keywords: [actionbar, selection, actions]
+after_version: 3.0.0-alpha.4
+---
+
+# ActionBar
+
+<PageDescription>{docs.exports.ActionBar.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionBar, ActionBarContainer, Item']}
+ />
+
+## Example
+```tsx example
+function StaticActionBar() {
+  return (
+    <ActionBarContainer>
+      <ActionBar
+        isEmphasized
+        selectedItemCount={3}>
+        <Item key="edit">
+          Edit
+        </Item>
+        <Item key="copy">
+          Copy
+        </Item>
+        <Item key="delete">
+          Delete
+        </Item>
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+<StaticActionBar />
+```
+
+## Content
+<Anatomy />
+
+An ActionBar is a companion component intended to facilitate bulk actions on selected items within a collection component. It consists of an ActionGroup, a clear button, and a count of selected items. It accepts a list of <TypeLink links={docs.links} type={docs.exports.Item} />
+elements as children, each with a `key` prop. Alternatively, a function that returns `Item` elements is also supported as seen below. See the [collection components](../react-stately/collections.html)
+docs for more details about this API. These `Item` elements are used to populate the ActionBar's ActionGroup and thus follows the [same content guidelines](./ActionGroup.html#content) when it comes to customization and labeling.
+
+```tsx example
+function DynamicActionBar() {
+  let barItems = [
+    {key: 'edit', label: 'Edit'},
+    {key: 'copy', label: 'Copy'},
+    {key: 'delete', label: 'Delete'}
+  ];
+
+  return (
+    <ActionBarContainer>
+      <ActionBar
+        items={barItems}
+        selectedItemCount={3}
+        isEmphasized>
+        {item => (
+          <Item key={item.key}>
+            {item.label}
+          </Item>
+        )}
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<DynamicActionBar />
+```
+
+### Layout
+ActionBarContainer accepts an ActionBar alongside any renderable children. It accepts the typical selection of [style props](#props), allowing you to customize its layout.
+The example below applies a static height on the ActionBarContainer instead of matching the height of the TableView.
```

Comment:
Should this sentence be more explicated that the TableView height is overridden by the ActionBarContainer height?

Reply:
to be honest, I'm not sure if that is the desired behavior now that I look at it again. Perhaps the TableView should preserve its own height?